<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gear Factory</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow_View"/>
   <addaction name="actionShow_Gear"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>&amp;View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="animateLabel">
         <property name="text">
          <string>Animate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="animateCheckBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="animationSpeedLabel">
         <property name="text">
          <string>Animation speed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="animationSpeedDoubleSpinBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="showGridLabel">
         <property name="text">
          <string>Show grid</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="showGridCheckBox">
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="unitLabel">
         <property name="text">
          <string>Unit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="unitComboBox">
         <item>
          <property name="text">
           <string>mm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cm</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="gridSizeLabel">
         <property name="text">
          <string>Grid size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="gridSizeDoubleSpinBox">
         <property name="suffix">
          <string>mm</string>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="showBaseCircleLabel">
         <property name="text">
          <string>Show base circle</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="showBaseCircleCheckBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="showExternalCircleLabel">
         <property name="text">
          <string>Show external circle</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="showExternalCircleCheckBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>E&amp;xport</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="fileNameLabel_2">
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="fileNameWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="formatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="formatComboBox">
         <item>
          <property name="text">
           <string>SVG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PNG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Gear</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="contactAngleLabel">
             <property name="text">
              <string>Contact angle</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="contactAngleDoubleSpinBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="toothSpacingLabel">
             <property name="text">
              <string>Tooth spacing</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="toothSpacingDoubleSpinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Wheel 1</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="numberOfTeethLabel">
             <property name="text">
              <string>Number of teeth</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="numberOfTeethSpinBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="baseDiameterLabel">
             <property name="text">
              <string>Base diameter</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="baseDiameterDoubleSpinBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="externalDiameterLabel">
             <property name="text">
              <string>External diameter</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="externalDiameterDoubleSpinBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="holeDiameterLabel">
             <property name="text">
              <string>Hole diameter</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="holeDiameterDoubleSpinBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="numberOfLighteningHolesLabel">
             <property name="text">
              <string>Number of lightening holes</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="numberOfLighteningHolesSpinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Wheel 2</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="numberOfTeethLabel_2">
             <property name="text">
              <string>Number of teeth</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="numberOfTeethSpinBox_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="baseDiameterLabel_2">
             <property name="text">
              <string>Base diameter</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="baseDiameterDoubleSpinBox_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="externalDiameterLabel_2">
             <property name="text">
              <string>External diameter</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="externalDiameterDoubleSpinBox_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="holeDiameterLabel_2">
             <property name="text">
              <string>Hole diameter</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="holeDiameterDoubleSpinBox_2"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="numberOfLighteningHolesLabel_2">
             <property name="text">
              <string>Number of lightening holes</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="numberOfLighteningHolesSpinBox_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionShow_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show View</string>
   </property>
   <property name="toolTip">
    <string>Show the 'view' window.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+V</string>
   </property>
  </action>
  <action name="actionShow_Gear">
   <property name="text">
    <string>Show Gear</string>
   </property>
   <property name="toolTip">
    <string>Show the 'gear' window.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+G</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Show the 'export' window.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
